src/composables/useSessionIssueHandler.js

```diff
@@ -46,6 +56,31 @@ export function useSessionIssueHandler() {
 		}
 	}
 
+	const handleSessionConflict = (token) => {
+		spawnDialog(ConfirmDialog, {
```

Comment:
Note: `spawnDialog` now supports Promise API. When a callback is not provided, it returns a promise resolved with the result

Reply:
Vue3 only.. I'll leave a FIXME comment